packages/flutter_tools/lib/src/flutter_plugins.dart

```diff
@@ -881,12 +919,21 @@ List<Plugin> _filterPluginsByVariant(List<Plugin> plugins, String platformKey, P
 
 @visibleForTesting
 Future<void> writeWindowsPluginFiles(FlutterProject project, List<Plugin> plugins, TemplateRenderer templateRenderer) async {
```

Comment:
we should consider how we're going to disentangle this in the world where platforms are not hardcoded into the tool AND templates are not hardcoded into the tool. For example, how does someone support FFI templates when creating their raspberry pi Flutter module?

cc @christopherfujino

Reply:
Yeah, this is a very good point that I hadn't considered until you brought it up.

Reply:
it might be an argument to suggest the ffi plugin templates should just stay in the tool.